### PR TITLE
fix(full-text-search): Fix PostgreSQL index example

### LINKED ISSUE #2216: full-text search: GIN index that's documented is wrong
Our current GIN index doesn't work. 

> Current FTS appear to not use defined GIN or GIST index
> 
> Recommended GIN index in doc is also not working
> 
> ERROR:  data type text has no default operator class for access method "gin"
> HINT:  You must specify an operator class for the index or define a default operator class for the data type.
> As far as I know it should be something like this
> 
> CREATE INDEX IF NOT EXISTS "post_body_index" ON "Posts" USING GIN (to_tsvector('english', "body"));
> -- Or/And
> CREATE INDEX IF NOT EXISTS "post_body_index" ON "Posts" USING GIST (to_tsvector('english', "body"));

Reference: https://github.com/prisma/prisma/issues/8950

### PR DESCRIPTION
**Pull request by janpio**
closes #2216

### CODE REVIEW

File: content/200-concepts/100-components/02-prisma-client/060-full-text-search.mdx [hunk 1]
```diff
@@ -82,18 +82,16 @@ Here's how the following queries would match that text:
...
 
-To speed up your full-text queries, you should add an index to your database.
-Prisma Migrate currently doesn't support adding search indices, but you can
-easily add one yourself.
+To speed up your full-text queries, you should add an index to your database. Prisma Schema currently doesn't support defining and Prisma Migrate doesn't support creating search indices, but you can easily add one yourself. The following example adds a `GIN` index to the table `posts` on the field `body`:
 
 ```sql
-CREATE INDEX post_body_index ON posts USING GIN (body);
+CREATE INDEX post_body_index 
+   ON posts USING GIN (to_tsvector('english', body));
```
Comment (ruheni) [inline content/200-concepts/100-components/02-prisma-client/060-full-text-search.mdx:179]: For a user setting up the index the first time, the Postgres extensions should be set up first.
```suggestion
CREATE EXTENSION pg_trgm;
CREATE EXTENSION btree_gin;
CREATE INDEX post_body_index 
   ON posts USING GIN (to_tsvector('english', body));
```

Comment (tanberry) [reply content/200-concepts/100-components/02-prisma-client/060-full-text-search.mdx:179]: Thanks @ruheni ! @janpio ,can we add Alex's section, and then merge?

Comment (janpio) [reply content/200-concepts/100-components/02-prisma-client/060-full-text-search.mdx:179]: Unfortunately not, as the PR got conflicted in the meantime (seems some other changes were merged to the same file) so I would ask you @tanberry to recreate the PR on the current page please and then just close this one.

Comment (ruheni) [reply content/200-concepts/100-components/02-prisma-client/060-full-text-search.mdx:179]: I've resolved the conflict. You can take another look @janpio if everything is all good.

File: content/200-concepts/100-components/02-prisma-client/060-full-text-search.mdx [hunk 2]
```diff
@@ -170,10 +170,13 @@ For the full range of supported operations, see the [MySQL full text search docu
 
 ### PostgreSQL
 
-To speed up your full-text queries, you should add an index to your database. Prisma Migrate currently doesn't support adding search indices in PostgreSQL, so this should be added in SQL. For example, the following SQL statement would add an index called `post_body_index` on the `body` column of the `posts` table:
+To speed up your full-text queries, you should add an index to your database. Prisma Schema currently doesn't support defining and Prisma Migrate doesn't support creating search indices, but you can easily add one yourself. The following example adds a `GIN` index to the table `posts` on the field `body`:
```
Comment (ruheni) [inline content/200-concepts/100-components/02-prisma-client/060-full-text-search.mdx]: The index would be created in the database, but the query executed/ generated by Prisma Client doesn't use the index. I'd prefer we wait till that issue is resolved in Prisma Client.

Related issue: https://github.com/prisma/prisma/issues/8950

Comment (janpio) [reply content/200-concepts/100-components/02-prisma-client/060-full-text-search.mdx]: The current state is the same though. Why should we wait on making the wording more exact about what we already document?

Comment (ruheni) [inline content/200-concepts/100-components/02-prisma-client/060-full-text-search.mdx]:
```suggestion
To speed up your full-text queries, you should add an index to your database. Prisma schema and Prisma Migrate currently don't support defining and creating PostgreSQL search indices, but you can add one yourself. The following example adds a `GIN` index to the table `posts` on the field `body`:
```
I hope the wording here is correct @janpio

**Review by ruheni — APPROVED** [review]
Everything looks okay. Once the wording is fixed, we can merge it

File: content/200-concepts/100-components/02-prisma-client/060-full-text-search.mdx (hunk 2 above)
Comment (janpio) [reply content/200-concepts/100-components/02-prisma-client/060-full-text-search.mdx]: Yes, the differentiation between "defining" and "creating" is probably not important enough to make the sentence that much harder to read than what I had.

**Pull request merged**